runtime/vm/createramclass.cpp

```diff
@@ -3589,26 +3589,28 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 			}
 		}
 		
-		if (J9_ARE_ALL_BITS_SET(romClass->modifiers, J9AccValueType)) {
-			UDATA instanceSize = result->totalInstanceSize;
+		if (J9ROMCLASS_IS_VALUE(romClass)) {
```

Comment:
I dont think we need this, we can just use `J9ROMCLASS_IS_PRIMITIVE_VALUE_TYPE`

Reply:
`J9ROMCLASS_IS_VALUE()` is here only to set `J9ClassIsValueType` on the ramClass so that `J9_IS_J9CLASS_VALUETYPE()` can continue working.

Reply:
ah okay, I missed that